moveit_ros/benchmarks/package.xml

```diff
@@ -1,13 +1,13 @@
 <package>
   <name>moveit_ros_benchmarks</name>
   <version>0.7.12</version>
-  <description>MoveIt tools for benchmarking</description>
 
-  <author email="isucan@google.com">Ioan Sucan</author>
-  <maintainer email="isucan@google.com">Ioan Sucan</maintainer>
-  <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
+  <description>Enhanced tools for benchmarks in MoveIt!</description>
 
-  <license>BSD</license>
```

Comment:
You accidentally removed the license tag here and CI complains about it...

Reply:
Thanks for the catch. Addressed. This is a cherrypick of a simpler change, so let me dismiss your review now that CI passed.